src/ontology/edits/oeo-physical.omn

```diff
@@ -10878,16 +10924,25 @@ Class: OEO_00030035
 Class: OEO_00050000
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "An industrial process is a process that has output object or object aggregates that are economic goods. An industrial process consists of several subprocesses. Examples of subprocesses that can be involved in an industrial process are energy transformations, mechanical operations and chemical reactions.",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A manufacturing process is a process that has produces artificial objects or object aggregates that are economic goods and that involves several subprocesses which are energy transformations and material transformations amongst others."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000118> "Herstellungsprozess"@de,
+        <http://purl.obolibrary.org/obo/IAO_0000118> "industrial process"@en,
```

Comment:
We should add the reference/mapping to https://spec.industrialontologies.org/iof/ontology/core/Core/ManufacturingProcess, e.g. via skos:closeMatch

Reply:
Thanks, I've added now in line 10938:
```
skos:closeMatch <https://spec.industrialontologies.org/ontology/core/Core/ManufacturingProcess>
```